app/team_test.go

```diff
@@ -67,7 +67,45 @@ func TestAddUserToTeam(t *testing.T) {
 	th := Setup(t).InitBasic()
 	defer th.TearDown()
 
+	t.Run("invite users by multiple domains", func(t *testing.T) {
```

Comment:
Why are these changes here? How are they connected to the plugin API?

Reply:
not sure how they got here :) will remove

Reply:
They still need to be removed :wink:

Reply:
I'm still unsure why these changes are there. Could you please explain?